src/studio/src/repositories/gitea-data/gitea/conf/app.ini

```diff
@@ -62,13 +48,13 @@ DISABLE_REGULAR_ORG_CREATION = true
 
 [security]
 INSTALL_LOCK   = true
-SECRET_KEY     = VCOEYZOz3vPZrm1vGBOL6paxSePtLUHc85LsK7QIqbPHRcohw72p5K8EQQJ7YrgA
-INTERNAL_TOKEN = eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYmYiOjE1MjMyNjM2NTR9.nxb5jknOl-LnVxOvyu0McHglWoixrfOD0mLZ8Druerw
 ;Use "off" to disable checking.
 PASSWORD_COMPLEXITY = lower,upper,digit,spec
 
 [mailer]
-SKIP_VERIFY     = false
+FROM        = studio@altinn.no
+MAILER_TYPE = smtp
+SKIP_VERIFY = false
 
 [service]
```

Comment:
Could not comment unchanged code, but as we are merking dev and staging/prod ini file we should opt to set defined values equal to the old production values and rather override them in dev
Line 63: `REGISTER_EMAIL_CONFIRM = false` should be `REGISTER_EMAIL_CONFIRM = true`

Reply:
Fixed.
Changed ENABLE_NOTIFY_MAIL to true as well.